modules/visual_script/visual_script_builtin_funcs.cpp

```diff
@@ -64,6 +64,7 @@ const char* VisualScriptBuiltinFunc::func_name[VisualScriptBuiltinFunc::FUNC_MAX
 	"str2var",
 	"var2bytes",
 	"bytes2var",
+	"color_named",
```

Comment:
I think it's a bad habit, as long as it compiles it can be left as is, but imagine you also add commas after last function param just because it's possible that you'll add an argument in future...
JSON forbids such syntax and I think it's good.
Commas should be used to separate elements and there is no element to be separated at the end.
Code should be as readible as possible and this -1, +2 is not a problem as review of such a change is trivial.
I should not use so many shoulds in my post. :)